### PR TITLE
Separate plots in examples w/ multiple plots using ###### 

### PR DESCRIPTION
**Pull request by ash13**
## PR Summary
Added ##### to programs which had more than 1 plots to seperate the graph plots at output.

As a part of this:
https://github.com/matplotlib/matplotlib/issues/8922

Fixed those files which were not already fixed from the list.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant 
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by jklymak** [comment]
Hi @ash13  The tests above indicate issues w/ the pull request.  If you can drill down in them, you should get hints as to whats wrong.  If you can't figure it out, feel free to ask!

**Comment by ash13** [comment]
sorry @jklymak ! I am quite new to contributing. Could you please help me figure what went wrong?

**Comment by dopplershift** [comment]
Two of the lines you added have > 80 characters, which fails our code style checks. According to the [sphinx gallery docs](https://sphinx-gallery.readthedocs.io/en/latest/tutorials/plot_notebook.html#sphx-glr-tutorials-plot-notebook-py) you only need a line of at least 20 '#' characters to make a new cell.

File: examples/axes_grid1/demo_fixed_size_axes.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def demo_fixed_size_axes():
 
     ax.plot([1, 2, 3])
 
-
+###########################################################################################
```
Comment (dstansby) [inline examples/axes_grid1/demo_fixed_size_axes.py]: The new lines need to be < 80 characters long (so ideally 79 characters!)

Comment (dstansby) [reply examples/axes_grid1/demo_fixed_size_axes.py]: (otherwise this PR looks good to me!)

**Comment by jklymak** [comment]
ping @ash13  I think this is close, but it needs a rebase and the various line lengths need to be fixed.  Drop a line if you can't sort it all out

**Comment by dstansby** [comment]
I'm going to close this since it's been dormant for a while, @ash13 leave a comment if you want it re-opened 😄

**Pull request closed without merging**